CriticalMass/RulesViewController.swift

```diff
@@ -31,17 +31,50 @@ enum Rule: String, CaseIterable {
 
 class RulesViewController: UITableViewController {
     private let rules = Rule.allCases
+    private let themeController: ThemeController!
```

Comment:
Is the explicit IUO needed since it is passed in the init?

Reply:
IUO? I need to get access to the currentTheme in `updateThemeIfNeeded()`, so I adapted the implementation from `MapViewController` and pass the themeController in the init.

Reply:
Sorry. I mean implicitly unwrapped optional
``` swift 
private let themeController: ThemeController
```
This should work, right?

Reply:
Ah of course! Yes, the implicitly unwrapped optional is not needed 👍 Will remove it